test/unit/api/threads.spec.ts

```diff
@@ -238,6 +240,50 @@ describe('Thread Tests', () => {
     });
   });
 
+  describe('/bulkThreads', () => {
+    it('should pass with public chain', async () => {
+      const res = await chai.request.agent(app)
+        .get('/api/bulkThreads')
+        .set('Accept', 'application/json')
+        .query({
+          chain,
+          jwt: adminJWT,
+        });
+      expect(res.body.result).to.not.be.null;
+      expect(res.body).to.not.be.null;
+      expect(res.body.status).to.be.equal('Success');
+    });
+    it('should pass with public community', async () => {
```

Comment:
I would change these descriptions to be more specific, saying the test should pass is not clear for a reader what this is testing.

Reply:
"should return bulk threads for a public chain"